app/client/components/membershipFAQs.tsx

```diff
@@ -22,184 +28,85 @@ export const MembershipFAQs = (props: RouteableProps) => (
           </div>
         </div>
-        <div title="What if I just want to give money to the Guardian?">
-          <div>
-            There is also the option to make a{" "}
-            <a
-              css={{
-                textDecoration: "underline",
-                color: palette.blue.dark,
-                ":visited": { color: palette.blue.dark }
-              }}
-              href="https://contribute.theguardian.com/?INTCMP=membership_faq"
-            >
-              contribution
-            </a>{" "}
-            if this is more suitable. All funds raised from readers will go
-            directly into supporting our journalism. We just ask for your name
-            and email address in order for you to do this (we need this in order
-            to process refunds and receipts, and in case of any transactional
-            issues).
-          </div>
-        </div>
-        <div title="Why don't you just have a paywall?">
+        <div title="Does membership provide an ad-free experience?">
           <div>
             <p>
-              We are currently running a series of tests asking people who
-              access theguardian.com if they would like to make a financial
-              contribution, either through becoming a Guardian Supporter or
-              through a one-off contribution. We hope that this will help
-              sustain our journalism and enable us to uncover the stories that
-              need to be told while – crucially – Guardian journalism remains
-              open to all.
+              As part of your membership, you qualify for free access to the
+              premium tier of the award-winning Guardian news app, which has no
+              adverts, allows you to follow your favourite journalists, sign up
+              for alerts to key topics, series and sports events, and gives you
+              access to our crosswords. You can download the app by visiting{" "}
+              <a css={linkCss} href="https://theguardian.com/guardianapp">
+                theguardian.com/guardianapp
+              </a>, and then you simply need to sign in with your Guardian
+              Members login details. If you encounter any difficulties accessing
+              the premium tier, please ensure you have the latest version of the
+              app downloaded. Once in the app you can choose between the UK, US,
+              Australia or International edition.
             </p>
-
             <p>
-              Of course a paywall is something we have looked at and we would
-              never rule it out but putting one up now would get in the way of
-              growing and deepening the relationships that are critical for our
-              future.
+              At the moment, only Digital Pack offers an ad-free experience on
+              the website.{" "}
+              <a
+                css={linkCss}
+                href="https://support.theguardian.com/uk/subscribe/digital"
```

Comment:
I think we should link to /subscribe/digital, rather than the UK version of this page.

Using the former allows us to redirect to the most appropriate version of the Digital Pack landing page based on a user's geolocation (see https://github.com/guardian/support-frontend/blob/master/conf/routes#L90-L91).

Reply:
Ah yes! Done.